src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -9153,7 +9125,7 @@ void VRFTerminalUnitEquipment::ControlVRFToLoad(EnergyPlusData &state,
     // set supplemental heating coil calculation if the condition requires
     if (this->SuppHeatingCoilPresent) {
         if (this->isSetPointControlled) {
-            auto &thisSuppHeatCoilAirInletNode = state.dataLoopNodes->Node(this->SuppHeatCoilAirInletNode);
+            auto const &thisSuppHeatCoilAirInletNode = state.dataLoopNodes->Node(this->SuppHeatCoilAirInletNode);
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:9156]:(style),[constVariable],Variable 'thisSuppHeatCoilAirInletNode' can be declared as reference to const